pkg/storage/unified/apistore/go.mod

```diff
@@ -15,11 +14,10 @@ require (
 	github.com/bwmarrin/snowflake v0.3.0
 	github.com/google/uuid v1.6.0
 	github.com/grafana/authlib/types v0.0.0-20250325095148-d6da9c164a7d
-	github.com/grafana/grafana v11.4.0-00010101000000-000000000000+incompatible
```

Comment:
^^^ removing this is the big change -- the rest are using the same version for grafana core all go.mod includes